domains/transfer/Transfer.API/API/UnitOfWork/IUnitOfWork.cs

```diff
@@ -30,38 +31,32 @@ public UnitOfWork(ApplicationDbContext context)
 
     public ITransferAgreementRepository TransferAgreementRepo
     {
-        get
-        {
-            return transferAgreementRepo ??= new TransferAgreementRepository(context);
-        }
+        get => transferAgreementRepo ??= new TransferAgreementRepository(context);
     }
 
     public IActivityLogEntryRepository ActivityLogEntryRepo
     {
-        get
-        {
-            return activityLogEntryRepo ??= new ActivityLogEntryRepository(context);
-        }
+        get => activityLogEntryRepo ??= new ActivityLogEntryRepository(context);
     }
 
     public ITransferAgreementProposalRepository TransferAgreementProposalRepo
     {
-        get
-        {
-            return transferAgreementProposalRepo ??= new TransferAgreementProposalRepository(context);
-        }
+        get => transferAgreementProposalRepo ??= new TransferAgreementProposalRepository(context);
     }
 
     public ITransferAgreementHistoryEntryRepository TransferAgreementHistoryEntryRepo
     {
-        get
-        {
-            return transferAgreementHistoryEntryRepo ??= new TransferAgreementHistoryEntryRepository(context);
-        }
+        get => transferAgreementHistoryEntryRepo ??= new TransferAgreementHistoryEntryRepository(context);
     }
 
     public Task SaveAsync()
     {
         return context.SaveChangesAsync();
     }
+
+    public async ValueTask DisposeAsync()
+    {
+        await context.DisposeAsync();
+        context = null!;
```

Comment:
Why set to null?

Reply:
removed: https://github.com/Energinet-DataHub/energy-origin/pull/950/commits/c15a735e902f4a51dea04005cc162286c4038966